jOOQ-checker/pom.xml

```diff
@@ -7,7 +7,7 @@
     <parent>
         <groupId>org.jooq</groupId>
         <artifactId>jooq-parent</artifactId>
-        <version>3.20.0-SNAPSHOT</version>
+        <version>3.20.1-SNAPSHOT</version>
```

Comment:
Why upgrade the version?

1. It's unrelated to your actual change
2. It's an incorrect update. 3.20.0 hasn't been released

Reply:
Ah I'm sorry, I didn't know. :)